01_fib/joel00wood/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func isWholeNumber(n float64) bool {
```

Comment:
I would suggest to use integers, thus you won't need this function, and it the ending program will be simpler as well

Reply:
Switched for integers as discussed and removed no longer necessary function